docs/core/setup/how-to-detect-installed-versions.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Check which .NET Core versions on Windows, Linux, and macOS
```

Comment:
```suggestion
title: Check which .NET Core versions on Windows, Linux, and macOS are installed - .NET Core
```

Reply:
Should we just add this folder to the `titleSuffix` entry in docfx.json?